terraform/modules/azure/review-app.tf

```diff
@@ -0,0 +1,6 @@
+resource "azurerm_resource_group" "review_app_rg" {
+  count    = var.create_env_resource_group ? 1 : 0
+  name     = var.resource_group_name
+  location = var.region_name
+  tags     = jsondecode(var.resource_group_tags)
```

Comment:
We can read the tags from the existing resource group via the data source: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/resource_group#tags

Reply:
Sorted